packages/versioning/src/versioning.ts

```diff
@@ -394,14 +394,15 @@ export function $useDependency(
     const targetEnum = target.enum;
     let state = context.program.stateMap(useDependencyEnumKey).get(targetEnum) as Map<
       EnumMember,
-      Version
+      Version[]
     >;
     if (!state) {
-      state = new Map<EnumMember, Version>();
-    }
-    for (const v of versions) {
-      state.set(target, v);
+      state = new Map<EnumMember, Version[]>();
     }
+    // get any existing versions and combine them
```

Comment:
Can we separate out these version changes to consider separately.  I don't think these are related to deprecations

Reply:
Yes, I already did that and it has been merged. This shouldn't show after the rebase.